src/azure-cli/azure/cli/command_modules/iot/tests/latest/test_iot_dps_commands.py

```diff
@@ -121,10 +121,10 @@ def test_dps_lifecycle(self, group_name, group_location):
         self.cmd('az iot dps certificate list --dps-name {} -g {}'.format(dps_name, group_name),
                  checks=[
                      self.check('length(value)', 2),
-                     self.check('value[0].name', cert_name_verified),
-                     self.check('value[0].properties.isVerified', True),
-                     self.check('value[1].name', cert_name),
-                     self.check('value[1].properties.isVerified', False)])
+                     self.check('value[1].name', cert_name_verified),
+                     self.check('value[1].properties.isVerified', True),
+                     self.check('value[0].name', cert_name),
+                     self.check('value[0].properties.isVerified', False)])
```

Comment:
This one has stumped me too - wondering if there's an ordering happening service-side that we may need to account for client-side, or perhaps we should just delete one of the certs before testing the second one.

Reply:
I am debating just doing a check that there are two certs and each cert is in the list (so we no longer care about the order)